components/Head.tsx

```diff
@@ -25,7 +25,7 @@ import NextHead from 'next/head';
 const PageHead = ({ subtitle }: { subtitle?: string }) => {
 	return (
 		<NextHead>
-			<title>Overture DMS{subtitle ? ` - ${subtitle}` : ''}</title>
+			<title>Overture {subtitle ? ` - ${subtitle}` : ''}</title>
```

Comment:
```suggestion
			<title>Overture Stage{subtitle ? ` - ${subtitle}` : ''}</title>
```

Reply:
I placed the stage title in the lab name, the title is a bit more hardcoded so I left it generic (meaning saving us changes needed on demo/quickstart)

Reply:
Sounds good. let's remove that trailing space then...
```suggestion
			<title>Overture{subtitle ? ` - ${subtitle}` : ''}</title>
```